test/tradeMe.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test'
+import { TMMainPage } from '../Pages/tMMainPage'
+
+test.describe('Tests Trademe car makes', () => {
+  let tmMainPage: TMMainPage
+
+  test.beforeEach(async ({ page }) => {
+    tmMainPage = new TMMainPage(page)
+    await tmMainPage.navigateToSite()
+  })
+
+  test('verify the car makes are displayed correctly', async ({ page }) => {
```

Comment:
There is a small issue with this code and is that, when executed, it won't show each car make as it's own test, but it will display a Given/Then streak of results one after the other. What you want to accomplish here is having one test, with one Given and Then, then another one (same test, different data), with the other car make. 

So, instead of:

Given the customer has selected the make "BMW"— tradeMe.spec.ts:21
Then the stock level for the "BMW" should be between 3101 and 3201.— tradeMe.spec.ts:25
Given the customer has selected the make "Mazda"— tradeMe.spec.ts:21
Then the stock level for the "Mazda" should be between 5928 and 6128.— tradeMe.spec.ts:25
Given the customer has selected the make "Honda"— tradeMe.spec.ts:21
Then the stock level for the "Honda" should be between 3322 and 3472.— tradeMe.spec.ts:25
Given the customer has selected the make "Ferrari"— tradeMe.spec.ts:21
Then the stock level for the "Ferrari" should be between 21 and 61.— tradeMe.spec.ts:25

It should be:
**verify the car makes are displayed correctly**
Given the customer has selected the make "BMW"— tradeMe.spec.ts:21
Then the stock level for the "BMW" should be between 3101 and 3201

**verify the car makes are displayed correctly**
Given the customer has selected the make "Mazda"— tradeMe.spec.ts:21
Then the stock level for the "Mazda" should be between 3101 and 3201

And so on. You can also add the car make dynamic bit in the name of the test to reflect which make is being tested in that iteration

Reply:
I will see if i can come up with something for this before i merge it.